services/drupal/web/modules/custom/epa_migrations/src/Plugin/migrate/process/EpaSetLatestRevision.php

```diff
@@ -140,7 +140,7 @@ public function transform($value, MigrateExecutableInterface $migrate_executable
     // current revision (as obtained from D7), retaining its same state.
     //
     // If they do match, skip this step and goto forward revision logic.
-    if ($node->vid !== $d7_vid) {
+    if ($node->get('vid')->value !== $d7_vid) {
```

Comment:
I'm surprised we need this change...did $node->vid really not behave as expected?

Reply:
`$node->vid` is returning a `FieldListItem` object. I found an entity API method for getting the vid and implemented it in my latest change: `$node->getRevisionId()`.